RiskAssessmentApi.yaml

```diff
@@ -248,9 +376,195 @@ components:
+                  type: number
+                  format: double
+                  minimum: 0.0
+                  maximum: 1.0
+                  description: The weighting of map tile availability risk
+              required:
+              - satelliteCount
+              - pdop
+              - mapTileAvailability
+            forceHighRiskIfMapTileUnavailable:
+              type: boolean
+              description: The toggle for forcing navigation reliability risk level to HIGH if map tile is unavailable
+          required:
+          - mediumThresholds
+          - highThresholds
+          - weights
+          - forceHighRiskIfMapTileUnavailable
+        overallRisk:
+          type: object
+          description: A collection of thresholds for determining overall risk level based on a weighting of the individual risk levels
+          properties:
+            weights:
+              type: object
+              description: The weighting to apply to each risk level when determining the overall risk. The weights should sum to 1.0
+              properties:
+                obstacleProximity:
+                  type: number
+                  format: double
+                  minimum: 0.0
+                  maximum: 1.0
+                  description: The weighting of obstacle proximity risk
+                groundImpact:
+                  type: number
+                  format: double
+                  minimum: 0.0
+                  maximum: 1.0
+                  description: The weighting of ground impact risk
+                navigationReliability:
+                  type: number
+                  format: double
+                  minimum: 0.0
+                  maximum: 1.0
+                  description: The weighting of navigation reliability risk
+              required:
+              - obstacleProximity
+              - groundImpact
+              - navigation
+            thresholds:
+              type: object
+              description: A collection of thresholds for determining overall risk level
+              properties:
+                medium:
+                  type: number
+                  format: double
+                  minimum: 0.0
+                  maximum: 1.0
+                  description: The threshold for setting overall risk level to MEDIUM
+                high:
+                  type: number
+                  format: double
+                  minimum: 0.0
+                  maximum: 1.0
+                  description: The threshold for setting overall risk level to HIGH
+              required:
+              - medium
+              - high
+            noGoHighRiskWaypointPercentage:
+              type: number
+              format: double
+              minimum: 0.0
+              maximum: 1.0
+              description: The threshold for the percentage of waypoints with an overall HIGH risk for setting Flight Go-Ahead Indicator to FALSE
+
     FlightOperation:
```

Comment:
So there is this FlightOperation object that includes waypoints, metadata and risk threshold information, but also the RiskAssessment object below contains a 'flightOperation' member that is a subset of this object (waypoints + uuid). 

This is structurally ok, but I'm wondering if we should deconflict the names to avoid confusion